pkg/azuredisk/azuredisk_v2.go

```diff
@@ -110,60 +114,121 @@ type DriverV2 struct {
 	azDriverNodeInformer              azdiskinformertypes.AzDriverNodeInformer
 	deviceChecker                     *deviceChecker
 	kubeClientQPS                     int
+	isControllerPlugin                bool
+	isNodePlugin                      bool
 }
 
 // NewDriver creates a driver object.
-func NewDriver(options *DriverOptions) CSIDriver {
-	return newDriverV2(options, *driverObjectNamespace, *nodePartition, *controllerPartition, *heartbeatFrequencyInSec, *controllerLeaseDurationInSec, *controllerLeaseRenewDeadlineInSec, *controllerLeaseRetryPeriodInSec, *leaderElectionNamespace)
+func NewDriver(config *azdiskv1beta2.AzDiskDriverConfiguration) CSIDriver {
+	getDriverConfig(config)
+	return newDriverV2(config)
+}
+
+func getDriverConfig(driverConfig *azdiskv1beta2.AzDiskDriverConfiguration) {
+	if *driverConfigPath != "" {
+		if driverConfig.ObjectNamespace == "" {
```

Comment:
We should initialize our defaults in a common place rather than split between `azuredisk_v2.go` and `azurediskplugin.go`.

Reply:
So just initialize all defaults together in `azurediskplugin/main.go`?

Reply:
Yes, that would be fine.